yoti-sdk/src/main/java/com/yoti/mobile/android/sdk/exceptions/AppNotInstalledErrorCode.java

```diff
@@ -0,0 +1,7 @@
+package com.yoti.mobile.android.sdk.exceptions;
+
+public enum AppNotInstalledErrorCode {
```

Comment:
I would call this enum as `AppType` so its values can store App types, and in `YotiSDKAppNotInstalledException` constructor, use this enum as param to clarify what's the app that is not found.

Reply:
This can also be used in the new listener to know what was the application that has been called: `https://github.com/getyoti/android-sdk-button/pull/21#discussion_r681615893

Imagine that one integrator has both, yoti and easyId buttons, so integrator can use same listener for both because listener events include the App type.

Reply:
Updated it into `AppType` in 5e1d4bf:) We are already providing which app is not installed in `YotiSDKAppNotInstalledException`.